notebooks/STIS/CoronagraphyViz/STIS_Coronagraphy_Visualization_v2.ipynb

```diff
@@ -0,0 +1,505 @@
+{
```

Comment:
<p>Based on its purpose, this function was designed to fit the requirements of this notebook, even though there may be some parts that appear to be duplicated. My suggestion is to leave it as it currently is. Additionally,&nbsp;the current version is more straightforward for readers who may not be familiar with programming.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>